[PoC RESULT]: False
[PoC MSG]:
============================== Run PoC ==============================
------------------------------ Standard Output ------------------------------
[Preload] Network requests are blocked, browser.js and sauce-test stubbed if present


  url-parse
    #set
      1) updates username and password when updating auth


  0 passing (7ms)
  1 failing

  1) url-parse
       #set
         updates username and password when updating auth:
     [1m[31mUnknown assertation failure occured, assumed `'user@'` to equal (===) `'user%40'`
  [90m/workspace/url-parse/test/test.js[0m
      [31m                                  v
      [90m1176.       assume(data.set('auth', 'user@:pass@')).equals(data);
      [0m1177.       assume(data.username).equals('user%40');
      [90m1178.       assume(data.password).equals('pass%40');
      [31m                                  ^[0m
          at Assume.equal (node_modules/assume/index.js:673:32)
          at Context.<anonymous> (test/test.js:1177:29)
          at process.processImmediate (node:internal/timers:491:21)
  [0m




------------------------------ Standard Error ------------------------------
(node:50) Warning: Accessing non-existent property 'then' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)

------------------------------ Finish Evaluation ------------------------------


[UnitTest RESULT]: None
[UnitTest MSG]:
 None

[Validation TYPE]: validation_fail